src/main/java/teammates/common/datatransfer/InstructorAttributes.java

```diff
@@ -311,4 +311,20 @@ public boolean isEqualToAnotherInstructor(InstructorAttributes instructor) {
                    && this.privileges.equals(instructor.privileges);
         }
     }
+    
+    public boolean isCustomRole() {
+        return (role != null) && role.equals(Const.InstructorPermissionRoleNames.INSTRUCTOR_PERMISSION_ROLE_CUSTOM);
```

Comment:
`Const.InstructorPermissionRoleNames.INSTRUCTOR_PERMISSION_ROLE_CUSTOM.equals(role)`
should give the same result.